public/index.html

```diff
@@ -4,8 +4,9 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="icon" href="<%= BASE_URL %>favicon.png">
```

Comment:
Does this work?

Reply:
It does but I switched back to ico . People tend to recommend to keep ico mainly because browsers look if there's a favicon.ico file on the root. And also because some browser did not support it before